hawc/apps/lit/templates/lit/_reference_with_tags.html

```diff
@@ -0,0 +1,82 @@
+<div id="tagged-ref-{{ref.pk}}">
+    {% if show_actions %}
+    <div class="dropdown btn-group float-right">
+        <a class="btn btn-sm btn-secondary dropdown-toggle" data-toggle="dropdown">Actions</a>
+        <div class="dropdown-menu dropdown-menu-right">
+            <a href="{% url 'lit:tag' assessment.pk%}?id={{ref.pk}}" class="dropdown-item" target="_blank"
```

Comment:
remove the `target="_blank" rel="noreferrer"` from these; move edit reference tags to be after edit reference